### ISSUE TITLE
[Bug]: When I create index for Milvus, Milvus hang.

### ISSUE THREAD

**Issue by JackLCL** [milvus-io/milvus#15705]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.0 GA
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):pymilvus 2.0.0
- OS(Ubuntu or CentOS): ubuntu
- CPU/Memory: 12cpu 64G
- GPU: 
- Others:
```

### Current Behavior

I follow this [link](https://milvus.io/docs/v2.0.0/install_cluster-helm.md) to start Milvus cluster.
After starting Milvus cluster, I run hello_milvus.py successfully.
I didn't do any operations for 4 days on this cluster.
Today I run hello_milvus.py again, but it hangs at creating an index.
![image](https://user-images.githubusercontent.com/53512883/155251009-f704f40b-7f49-4003-ab51-b3cc3ffa8e7d.png)

All pods in cluster look well:
![image](https://user-images.githubusercontent.com/53512883/155251099-7f3e5146-ffc5-4a80-8d82-f11b82baa09c.png)

### Expected Behavior

I can run hello_milvus.py successfully.

### Steps To Reproduce

```markdown
1. start Milvus cluster 
2. don't do anything for 4 days
3. run hello_milvus.py
```

**Comment by yanliang567** [comment]
@JackLCL please upload the logs for milvus pods

**Comment by JackLCL** [comment]
@yanliang567 
[milvus_log.zip](https://github.com/milvus-io/milvus/files/8121611/milvus_log.zip)

**Comment by xiaofan-luan** [comment]
@DragonDriver any progress?

**Comment by DragonDriver** [comment]
I'm so sorry for the late reply. It seems that the log of indexcoord is not complete, I can't see any index-related logs in indexcoord. Apologize again that I didn't focus on this issue. @JackLCL

**Comment by DragonDriver** [comment]
```shell
$ cat indexcoord.log | wc -l
45100
$ cat indexcoord.log | grep "watchMetaLoop" | wc -l
45100
```

**Comment by learningpro** [comment]
why close this issue, I faced this bug as well. It's not hard to reproduce this issue, just do as your document says  https://milvus.io/docs/v2.1.x/example_code.md

**Comment by ThomasAlxDmy** [comment]
Same here, ran into this issue. It seems like sometimes queries hang forever even given some client timeouts. Would be good to understand what's going on and see errors.

**Comment by yanliang567** [comment]
this issue was verified fixed on 2.2, please retry on 2.2.2, and if it reproduced to you, feel free to file a new issue with full milvus logs.

**Comment by LoriZhuka28** [comment]
I got the same issue
